test/test_python_ex_template.py

```diff
@@ -0,0 +1,44 @@
+import os
+import sys
+import pytest
+from pathlib import Path
+
+from container_ci_suite.openshift import OpenShiftAPI
+from container_ci_suite.utils import get_service_image, check_variables
+test_dir = Path(os.path.abspath(os.path.dirname(__file__)))
+
+if not check_variables():
+    sys.exit(1)
```

Comment:
What about some error message here is the exit code is non-zero?

Reply:
The messages are here: https://github.com/sclorg/container-ci-suite/blob/main/container_ci_suite/utils.py#L242.

Reply:
Done